src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -170,6 +170,7 @@ class DDIMScheduler(SchedulerMixin, ConfigMixin):
 
     _compatibles = [e.name for e in KarrasDiffusionSchedulers]
```

Comment:
Could we create a new scheduler class here, `DDIMParallelScheduler`?

I'd like to keep the DDIM scheduler simple if possible :-)

Reply:
ok done!